lisa/tools/lspci.py

```diff
@@ -68,10 +75,28 @@ def __str__(self) -> str:
     def parse(self, raw_str: str) -> None:
         matched_pci_device_info = PATTERN_PCI_DEVICE.match(raw_str)
```

Comment:
Use LISA built-in regexp wrappers, it will save lines on matches.

Reply:
@squirrelsc 

This line was not part for this code change and been this way since beginning of this work. Do you want me to refactor this too.

Reply:
Yes, do refactoring to make code life longer.

Reply:
@squirrelsc Used method provided by `lisa.utils` now. Please check the latest push